### DISCUSSION TITLE
How to change the app icon on windows?

### DISCUSSION THREAD

**Discussion by fayfive** [fyne-io/fyne#3238] [Q&A] [answered]
How to change the app icon on windows?

**Comment by Jacalz** [comment, marked as answer]
You need to set your icon using `a.SetIcon()` (assuming `a` is your application instance) as usual. However, you need to install the `fyne.io/fyne/v2/cmd/fyne` command and then package the application using the `fyne package` subcommand.

More information can be found at https://developer.fyne.io/started/packaging. The developer site is a great place to find developer related information and documentation about Fyne.
